waspc/src/Wasp/Analyzer/TypeChecker/TypeError.hs

```diff
@@ -82,22 +82,35 @@ data TypeCoercionErrorReason e
     ReasonDictWrongKeyType String e
   deriving (Eq, Show)
 
-getTypeCoercionErrorMessageAndCtx :: (Type -> TypedExpr -> String) -> TypeCoercionError -> (String, Ctx)
-getTypeCoercionErrorMessageAndCtx getUncoercableTypesMsg (TypeCoercionError (WithCtx ctx texpr) t reason) =
+second :: (b -> d) -> (a, b, c) -> (a, d, c)
+second f (x, y, z) = (x, f y, z)
```

Comment:
This should most likely be in `Wasp.Util`. Should I even be implementing it or is it a code smell (`second` from `Control.Arrow` only works on pairs).